hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncConnectionImpl.java

```diff
@@ -181,9 +185,18 @@ public void newDead(ServerName sn) {
     this.clusterStatusListener = listener;
   }
 
+  public void startChoreService() {
```

Comment:
I do not think we need this method? Just do the initialization work in the getChoreService method?